app/assets/about.md

```diff
@@ -127,3 +127,16 @@ stateCases = tryCatch(
 ```
   
   
+
+##### Forecast evaluation
+
+If you are interested in getting the forecasts paired with the corresponding actual values (if you were e.g. testing different evaluation methods), that can be found at [https://www.stat.berkeley.edu/~ryantibs/data/](https://www.stat.berkeley.edu/~ryantibs/data/ "this is a temporary home") in 3 (zipped) zip files.
+These files are static, and the latest forecast date is
```

Comment:
suggestion: Be a little more specific about generation, maybe

```suggestion
These files are static, generated using [the aggregation script](link TBD), and forecast and actual data available on June 12, 2023. The latest forecast date available for each target signal is
```

Waiting on Ryan to share his code.